### PR TITLE
LPS-121417 baseline

### PR DESCRIPTION
**Pull request by marcosapmf**
Related Issue: https://issues.liferay.com/browse/LPS-121417

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hey @marcosapmf, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @marcosapmf, can you please check if these failures are related to your changes? Thanks

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @marcosapmf, can you please check if these failures are related to your changes? Thanks

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @marcosapmf, can you please check if these failures are related to your changes? Thanks

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/DDMFormEvaluatorEvaluateRequest.java [hunk 1]
```diff
@@ -41,6 +41,10 @@ public DDMFormValues getDDMFormValues() {
 		return _ddmFormValues;
 	}
 
+	public boolean getEditing() {
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/DDMFormEvaluatorEvaluateRequest.java:44]: @marcosapmf, maybe would be better name it `isEditing`.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/DDMFormEvaluatorEvaluateRequest.java [hunk 2]
```diff
@@ -110,6 +120,7 @@ private DDMFormEvaluatorEvaluateRequest() {
 	private DDMForm _ddmForm;
 	private DDMFormLayout _ddmFormLayout;
 	private DDMFormValues _ddmFormValues;
+	private boolean _editing;
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/DDMFormEvaluatorEvaluateRequest.java:123]: @marcosapmf, what exactly is this `editing` attribute referring?

**Review by natocesarrego — COMMENTED** [review]
Good job, @marcosampf! Please see my suggestions below.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/DDMFormEvaluatorEvaluateRequest.java (hunk 2 above)
Comment (natocesarrego) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/DDMFormEvaluatorEvaluateRequest.java:123]: @marcosapmf, maybe we should rename it to `_userEditing`, after all the user is editing the field and the field is being edited, what do you think?

Comment (marcosapmf) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/DDMFormEvaluatorEvaluateRequest.java:123]: @natocesarrego it storages the information if the user is editing a field. Maybe we should rename it to `_editingField`. What do you think?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/numeric/NumericDDMFormFieldValueLocalizer.java [hunk 3]
```diff
@@ -55,6 +56,16 @@ public String localize(String value, Locale locale) {
 				number = defaultDecimalFormat.parse(value);
 
 				formattedNumber = decimalFormat.format(number);
+
+				String lastChar = String.valueOf(
+					value.charAt(value.length() - 1));
+
+				if (_editing &&
```
[the remark this answers is not in the document]
Comment (marcosapmf) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/numeric/NumericDDMFormFieldValueLocalizer.java:63]: @natocesarrego, it sounds nice. I'll do that

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**